frontend-react/src/lib/widget/component/error-fallback.module.css

```diff
@@ -0,0 +1,5 @@
+.alert {
+	max-height: 100%;
+	min-height: 100%;
+	overflow-y: auto;
```

Comment:
Have to revisit this – styling of the widgets (and error fallbacks) needs some more work, but can be handled outside this PR.